.github/workflows/tests.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
I had to add that because BC checker requires to access all history for all branches and tags.